collections/impact-observatory-lulc-map.yaml

```diff
@@ -1,14 +1,14 @@
 Name: 10m Annual Land Use Land Cover (9-class) 
 Description: |
-  The 10m Annual Land Use Land Cover (LULC) map is produced Impact Observatocy, Microsoft, and Esri collaboratively.
+  The [10m Annual Land Use Land Cover (LULC)](https://www.impactobservatory.com/global_maps/) map is produced Impact Observatocy, Microsoft, and Esri collaboratively.
   The data collection is derived from ESA Sentinel-2 imagery at 10m resolution globaly using Impact Ovservatory's state of the art deep learning AI land classification model which is trained by billions of human-labeled image pixels.
   There are 9 LULC classes generated by the algorithm, including Built, Crops, Trees, Water, Rangeland, Flooded Vegetation, Snow/Ice, Bare Ground, and Clouds.
 
 AdditionalInfoExternal:
     Title: Additional info
     Path: impact-observatory-lulc-map/README.MD 
-Image: impact-observatory-lulc-map/impact-observatory-lulc-map.png
-ViewOnEOBrowser: 
+Image: impact-observatory-lulc-map/io-lulc-map.png
```

Comment:
I suggest not to use this image as thumbnail also, it fits well as the representative image

but rather find a nice location that shows the landcover  and get a screen shot of a square size for the thumbnail